recipes/bigdft/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "bigdft-suite" %}
+{% set version = "1.9.3_rc.2" %}
+{% set build = 0 %}
+{% set mpi = mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/l_sim/bigdft-suite/-/archive/{{ version }}/bigdft-suite-{{ version }}.tar.gz
+  md5: a756c1673c45e7e7b7be3cdf6aef81b1
+  patches:
+    - patches/output.patch
+    - patches/utils.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - autoconf
+    - automake
+    - binutils
+    - make
+    - pip
+  host:
+    - {{ mpi }}
+    - libblas
+    - liblapack
+    - yaml
+    - python
+    - libxc <5.0
+    - ntpoly
+    - setuptools
+  run:
+    - {{ mpi }}
+    - python
+    - scipy
+    - numpy
+    - pyyaml
+    - libxc <5.0
+    - openssh  # [mpi == 'openmpi']
+
+test:
+  files:
+    - test.py
+
+about:
+  home: https://bigdft.org
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
```

The files in licenses are not being included, but I think globbing works? otherwise, you need to be explicit.

```suggestion
  license_file:
    - LICENSE
    - licenses/*
```